reference/library/src/morpheus-master/js/src/sakai.morpheus.xlogin.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * For xlogin changes
+ */
+
+/* Show server time on footer if container exists */
+
+// Chris Coyier's MD5 Library
+// http://css-tricks.com/snippets/javascript/javascript-md5/
+var MD5 = function (string) {
```

Comment:
The file is xlogin but the main bit is an MD5 impl. Can we make this a bit more generic in case others want to use the same MD5 function? If not, all good.